--- libtransmission/web.c.save	2019-08-13 09:42:43.000000000 +0800
+++ libtransmission/web.c	2019-08-13 09:41:58.000000000 +0800
@@ -145,8 +145,8 @@
   /* announce and scrape requests have tiny payloads. */
   if (isScrape || isAnnounce)
     {
-      const int sndbuf = isScrape ? 4096 : 1024;
-      const int rcvbuf = isScrape ? 4096 : 3072;
+      const int sndbuf = 8192;
+      const int rcvbuf = 8192;
       setsockopt (fd, SOL_SOCKET, SO_SNDBUF, (const void *) &sndbuf, sizeof (sndbuf));
       setsockopt (fd, SOL_SOCKET, SO_RCVBUF, (const void *) &rcvbuf, sizeof (rcvbuf));
     }
